Demo npm run dev fails

Here is some early feedback. Hopefully it helps yall out...

C:\2017\jsapi-resources-master\4.x\typescript\demo>npm install
demo@1.0.0 C:\2017\jsapi-resources-master\4.x\typescript\demo
+-- @types/arcgis-js-api@4.2.0
+-- tslint@4.3.1
| +-- babel-code-frame@6.20.0
| | +-- chalk@1.1.3
| | | +-- ansi-styles@2.2.1
| | | +-- escape-string-regexp@1.0.5
| | | +-- has-ansi@2.0.0
| | | | `-- ansi-regex@2.1.1
| | | +-- strip-ansi@3.0.1
| | | `-- supports-color@2.0.0
| | +-- esutils@2.0.2
| | `-- js-tokens@2.0.0
| +-- colors@1.1.2
| +-- diff@3.2.0
| +-- findup-sync@0.3.0
| | `-- glob@5.0.15
| +-- glob@7.1.1
| | +-- fs.realpath@1.0.0
| | +-- inflight@1.0.6
| | | `-- wrappy@1.0.2
| | +-- inherits@2.0.3
| | +-- minimatch@3.0.3
| | | `-- brace-expansion@1.1.6
| | |   +-- balanced-match@0.4.2
| | |   `-- concat-map@0.0.1
| | +-- once@1.4.0
| | `-- path-is-absolute@1.0.1
| +-- optimist@0.6.1
| | +-- minimist@0.0.10
| | `-- wordwrap@0.0.3
| +-- resolve@1.2.0
| +-- underscore.string@3.3.4
| | +-- sprintf-js@1.0.3
| | `-- util-deprecate@1.0.2
| `-- update-notifier@1.0.3
|   +-- boxen@0.6.0
|   | +-- ansi-align@1.1.0
|   | +-- camelcase@2.1.1
|   | +-- cli-boxes@1.0.0
|   | +-- filled-array@1.1.0
|   | +-- object-assign@4.1.1
|   | +-- repeating@2.0.1
|   | | `-- is-finite@1.0.2
|   | |   `-- number-is-nan@1.0.1
|   | +-- string-width@1.0.2
|   | | +-- code-point-at@1.1.0
|   | | `-- is-fullwidth-code-point@1.0.0
|   | `-- widest-line@1.0.0
|   +-- configstore@2.1.0
|   | +-- dot-prop@3.0.0
|   | | `-- is-obj@1.0.1
|   | +-- graceful-fs@4.1.11
|   | +-- mkdirp@0.5.1
|   | | `-- minimist@0.0.8
|   | +-- os-tmpdir@1.0.2
|   | +-- osenv@0.1.4
|   | +-- uuid@2.0.3
|   | `-- write-file-atomic@1.3.1
|   |   +-- imurmurhash@0.1.4
|   |   `-- slide@1.1.6
|   +-- is-npm@1.0.0
|   +-- latest-version@2.0.0
|   | `-- package-json@2.4.0
|   |   +-- got@5.7.1
|   |   | +-- create-error-class@3.0.2
|   |   | | `-- capture-stack-trace@1.0.0
|   |   | +-- duplexer2@0.1.4
|   |   | +-- is-redirect@1.0.0
|   |   | +-- is-retry-allowed@1.1.0
|   |   | +-- is-stream@1.1.0
|   |   | +-- lowercase-keys@1.0.0
|   |   | +-- node-status-codes@1.0.0
|   |   | +-- parse-json@2.2.0
|   |   | | `-- error-ex@1.3.0
|   |   | |   `-- is-arrayish@0.2.1
|   |   | +-- pinkie-promise@2.0.1
|   |   | | `-- pinkie@2.0.4
|   |   | +-- read-all-stream@3.1.0
|   |   | +-- readable-stream@2.2.2
|   |   | | +-- buffer-shims@1.0.0
|   |   | | +-- core-util-is@1.0.2
|   |   | | +-- isarray@1.0.0
|   |   | | +-- process-nextick-args@1.0.7
|   |   | | `-- string_decoder@0.10.31
|   |   | +-- timed-out@3.1.3
|   |   | +-- unzip-response@1.0.2
|   |   | `-- url-parse-lax@1.0.0
|   |   |   `-- prepend-http@1.0.4
|   |   +-- registry-auth-token@3.1.0
|   |   | `-- rc@1.1.6
|   |   |   +-- deep-extend@0.4.1
|   |   |   +-- ini@1.3.4
|   |   |   +-- minimist@1.2.0
|   |   |   `-- strip-json-comments@1.0.4
|   |   `-- registry-url@3.1.0
|   +-- lazy-req@1.1.0
|   +-- semver-diff@2.1.0
|   | `-- semver@5.3.0
|   `-- xdg-basedir@2.0.0
|     `-- os-homedir@1.0.2
`-- typescript@2.1.5

npm WARN demo@1.0.0 No repository field.
npm WARN demo@1.0.0 license should be a valid SPDX license expression

C:\2017\jsapi-resources-master\4.x\typescript\demo>npm run dev

> demo@1.0.0 dev C:\2017\jsapi-resources-master\4.x\typescript\demo
> tsc -w .

**error TS6053: File '.ts' not found.**
4:26:12 PM - Compilation complete. Watching for file changes.

--- a/4.x/typescript/demo/README.md
+++ b/4.x/typescript/demo/README.md
@@ -1,8 +1,10 @@
-# Demo App for using TypeScript with [ArcGIS API 4 for JavaScript](https://developers.arcgis.com/javascript/)
+# Demo App
+
+> TypeScript with the [ArcGIS API 4 for JavaScript](https://developers.arcgis.com/javascript/)
 
 ## Usage
 
-Clone the repo and run `npm install`.
-
-* `npm run dev` - Compile TypeScript and watch for file changes.
-* `npm run lint` - Run TSLint on TypeScript files.
+1. Clone the repo and `npm install` dependencies
+2. `npm run dev` to compile `/app/main.ts` to `main.js` in the same folder and watch for changes
+3. launch `index.html` in your browser of choice.
+4. `npm run lint` to run [TSLint](https://github.com/palantir/tslint) on TypeScript files.

--- a/4.x/typescript/demo/package.json
+++ b/4.x/typescript/demo/package.json
@@ -2,7 +2,10 @@
   "name": "demo",
   "version": "1.0.0",
   "description": "Demo application using TypeScript with ArcGIS API 4 for JavaScript",
-  "repository": "https://github.com/Esri/jsapi-resources/tree/master/4.x/typescript/demo",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Esri/jsapi-resources.git"
+  },
   "scripts": {
     "dev": "tsc -w",
     "lint": "tslint './app/*.ts'"

